ui/app/templates/vault/cluster/access/identity/index.hbs

```diff
@@ -21,7 +21,18 @@
               glyph="role"
               size=14
               class="has-text-grey-light"
-            }}<span class="has-text-weight-semibold">{{item.id}}</span></a>
+            }}<span class="has-text-weight-semibold">{{item.name}}</span></a>
+          <div class="has-text-grey is-size-8">
+            {{item.id}}
+          </div>
+        </div>
+        <div class="column is-3 is-hidden-mobile">
+          {{#if (eq item.identityType "entity")}}
+            {{#if item.aliases.length}}
+              {{pluralize item.aliases.length "alias"}}
+            {{/if}}
+          {{else}}
```

Comment:
✂️ ?

Reply:
Oh left that in from groups, can take out the outer if entirely until we land other groups stuff.